README.md

```diff
@@ -12,4 +12,12 @@ Highly Available Tomcat 9 Image
        -> Update your application build.yaml to use this image for hosting your java application 
        -> Update your application deployment.yaml to add the rbac authorizations and roles required to run the tomcat in a cluster and don't forget to add the service account required to run your container under the spec.
 
+## Build & Deploy: Tag Convention
+    We build in perrsi-tools. Per the Red Hat standard, we will use the time since the epoch as a tag suffix to the Tomcat version:
+        > oc process -f ./openshift/build.yaml -p TAG_SUFFIX=$(date +%s) -p SOURCE_GIT_REF=release/0.0.1 | oc create -n perrsi-tools -f -
```

Comment:
It may be worthwhile to mention that these commands are being run from a developers terminal. The comment seem like suggestions to use '''oc replace'''. So does it mean some part of the pipeline has to change to do oc replace?

Reply:
I added a mention for this ('via oc CLI'); not sure what you mean about the pipeline, but this is all for manual building of ha-tomcat9 images by oneteamers. None of this applies to consumers of the image.